Sketch.java

```diff
@@ -16,21 +16,172 @@ public void settings() {
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(66, 228, 253);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
+ 
+   // All drawing methods
   public void draw() {
```

Comment:
missing [level 4 requirement](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-4), "demonstration of method with parameters and a return value. i.e methods such that given dimensions of an object, returns the x value or y value for the object to be drawn in the middle of the screen."